docs/guides/es2k/building-p4cp-acc.md

```diff
@@ -0,0 +1,132 @@
+# Building P4 Control Plane for the ACC
+
+This document explains how to build P4 Control Plane for the ARM Compute
+Complex (ACC) of the Intel&reg; IPU E2100.
+
+## Prepare the System
+
+There are several things to do before you can build P4 Control Plane.
+
+- Install CMake 3.15 or above
+
+  Avoid versions 3.24 and 3.25. They cause the dependencies build to fail.
+
+- Install OpenSSL 1.1
+
+  Note that P4 Control Plane is not compatible with BoringSSL.
+
+- Install the ACC SDK
+
+  See [Installing the ACC SDK](installing-acc-sdk.md) for instructions.
+
+## Install Stratum Dependencies
+
+See [Building Stratum Dependencies for the ACC](../../../setup/building-es2k-stratum-deps.md)
+for instructions.
+
+You will need the Host depencies, Target dependencies, and Build environment
+setup file, in order to cross-compile P4 Control Plane for the ACC.
+
+## Fetch Source Code
+
+If you have not already done so, you will need to get a copy of the IPDK
+networking-recipe repository and its submodules:
+
+```bash
+git clone --recursive https://github.com/ipdk-io/networking-recipe.git ipdk.recipe
+```
+
+You may substitute your own local directory name for `ipdk.recipe`.
+
+## Build with OVS
+
+### Setup
+
+First, change to the source directory:
+
+```bash
+cd ipdk.recipe
+```
+
+If you have not already done so, source the file that the defines the
+[target build environment variables](../../../setup/building-es2k-stratum-deps.md#5-defining-the-target-build-environment).
+For example:
+
+```bash
+source setup/es2k-setup.env
```

Comment:
This example is misleading, user might think this is the command and reports there is no file named "es2k-setup.env"

Reply:
I'll try expanding the explanation in the lead-in paragraph.

Reply:
I wound up changing the example to "command with placeholder", followed by an explanation.